### PR TITLE
Change akka version to 2.5.3

### PR DESCRIPTION
**Pull request by uandrew1965**

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @uandrew1965,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Comment by jrudolph** [comment]
Hi @uandrew1965, thanks for the PR.

We have not yet decided how to tackle compiling against 2.5 exactly. See #1151.

File: .travis.yml [hunk 1]
```diff
@@ -3,7 +3,8 @@ language: scala
 sudo: false
 
 scala:
-  - "2.11.8"
+  - "2.11.11"
+  - "2.12.2"
```
Comment (raboof) [inline .travis.yml]: Travis is currently used just for whitesource and publishing, and publishing is done 'in one go' by `+publish`, so the 2 different Scala versions are not needed here. I'll update `CONTRIBUTING.MD` to make this a bit more clear.

Comment (raboof) [inline .travis.yml]: https://github.com/akka/akka-http/pull/1283

**Comment by jrudolph** [comment]
I'm closing this for now, not because we don't want to make this change, but because we have to think about how to test this properly etc.

**Pull request closed without merging**